perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/TasksManagerBlImpl.java

```diff
@@ -41,6 +41,8 @@
 	@Autowired
 	protected TasksManagerImplApi tasksManagerImpl;
 
+	private static boolean suspendedTasksPropagation = false; //are tasks stopped from propagating to engine
```

Comment:
We should have safe concurrent access here. You can use `AtomicBoolean` or you can synchronize all access to the variable using `synchronized(TasksManagerBlImpl.class) { ...read/write code.... }` in getter/setter.